script/Ciel/Day 14/14_01B_02_CIEL14B_2B.txt

```diff
@@ -1,107 +1,119 @@
 [sha:b73f8c1cb1bc996f9a43bfffef7bea5552229afe]{
 -- Page 10001, Offset 39436.
 -- ……こんな光景を目にする時が来るとは思わなかった。
-...I never thought I would see a scene like this.
+Never in a million years would I have imagined I'd see something like this...
 }
 [sha:89946ea1caf734908b4d76555a6895194d076ec7]{
 -- Page 10001, Offset 39437.
 -- アレも<落盤|らくばん>と言っていいのだろうか。
-I wonder if you could even call that a cave-in.
+Can you even call something like that a 'cave-in' at this point?
```

Comment:
Landslide doesn't quite sit right with me; feels too slumpy / unseismic. Maybe get quakey?
> I don't think you could call this a simple earthquake at this point.
> Could you even consider what's happening right now an earthquake?

Reply:
Hmmm, I think referring to it as a fissure might be prudent. Earthquake gives another vibe entirely, and landslide is a lot about the sliding part rather than the crack which causes the slide. Helps that it's referred to as a fissure in the last file.